geometry_msgs/Pose pose
---
uint32 loc_id
